talk/owt/sdk/base/mediautils.cc

```diff
@@ -9,28 +9,36 @@
 #include "webrtc/common_video/h264/prefix_parser.h"
 #include "webrtc/rtc_base/checks.h"
 #include "webrtc/rtc_base/bit_buffer.h"
+#include "webrtc/rtc_base/string_to_number.h"
 #include "system_wrappers/include/field_trial.h"
 #include "talk/owt/sdk/base/mediautils.h"
+#include "talk/owt/sdk/base/win/mediacapabilities.h"
 
 namespace owt {
 namespace base {
+
 static const std::map<const std::string, const Resolution> resolution_name_map = {
     {"cif", Resolution(352, 288)},
     {"vga", Resolution(640, 480)},
     {"hd720p", Resolution(1280, 720)},
-    {"hd1080p", Resolution(1920, 1080)}};
+    {"hd1080p", Resolution(1920, 1080)},
```

Comment:
Anywhere else is still using strings(hd720p, vga) to represent resolutions?

Reply:
I believe QA's test application is using that.